pkg/machinery/extensions/extensions.go

```diff
@@ -10,12 +10,12 @@ import "path/filepath"
 // AllowedPaths lists paths allowed in the extension images.
 var AllowedPaths = []string{
 	"/etc/cri/conf.d",
-	"/lib/firmware",
-	"/lib/modules",
+	"/usr/lib/firmware",
+	"/usr/lib/modules",
 	// The glibc loader is required by glibc dynamic binaries.
-	"/lib64/ld-linux-x86-64.so.2",
-	// /sbin/ldconfig is required by the nvidia container toolkit.
-	"/sbin/ldconfig",
+	"/usr/lib/ld-linux-x86-64.so.2",
+	// /usr/bin/ldconfig is required by the nvidia container toolkit.
+	"/usr/bin/ldconfig",
```

Comment:
@frezbo ?

Reply:
na, we can't do this, it neds to be `/sbin/ldconfig` which is standard expected for glibc

Reply:
the idea is that in runtime we have a symlink, e.g. `/sbin` -> `/usr/bin/ldconfig`

Reply:
hmm that should work then, but I guess extensions should still allow the path :thinking: